data/data/bootstrap/baremetal/systemd/units/wait-iptables-init.service.template

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Wait for iptables to be initialised
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+# Create a symlink to the correct iptables binary for the provisioning network
+ExecStartPre=ln -fs /usr/sbin/ip{{if .PlatformData.BareMetal.ProvisioningIPv6}}6{{end}}tables /usr/local/bin/prov-iptables
```

Comment:
OMG, please no. Let's create a templated script like dhcp-filter.sh.

Reply:
I don't see creating scripts in runtime as a good idea, especially if some other scripts are created in advance.

Reply:
Even worse, the inspector service implicitly depends on this script...

Reply:
Done.